### ISSUE TITLE
Consumables UI cutoff

### ISSUE THREAD

**Issue by RaySolo137** [DestinyItemManager/DIM#8195]
### DIM Version

Version 7.8.3 (release), built on 3/16/2022, 9:24:24 AM

### Browser Details

Chrome Version 99.0.4844.74 (Official Build) (64-bit)

### OS Details

Windows 11

### Describe the bug

UI for consumables is getting cutoff. I'm able to see only few details. Pasting the screenshot below.

![image](https://user-images.githubusercontent.com/55454865/159150952-6d9092f2-2e08-4c1f-b9c6-9a1f4e5c8ef3.png)

### Logs

```shell
instrument.ts:125 [app] DIM v7.8.3 (release) - Please report any errors to https://www.github.com/DestinyItemManager/DIM/issues
instrument.ts:125 Language Detected: en-us
instrument.ts:125 Language Assigned: en
hooks.ts:53 [Intervention] Slow network is detected. See https://www.chromestatus.com/feature/5636954674692096 for more details. Fallback font will be used while loading: https://fonts.gstatic.com/s/opensans/v28/memvYaGs126MiZpBA-UvWbX2vVnXBbObj2OVTS-muw.woff2
instrument.ts:125 [dim sync] globalSettings Object
log.ts:58 Load manifest: 2649.572998046875 ms
instrument.ts:125 [dim sync] Saving profile data to IDB
log.ts:58 Process inventory: 193.68603515625 ms
log.ts:58 Inventory state update: 971.22119140625 ms
log.ts:58 Process inventory: 163.232177734375 ms
log.ts:58 Inventory state update: 348.259033203125 ms
DevTools failed to load source map: Could not load content for chrome-extension://eppiocemhmnlbhjplcgkofciiegomcon/content/safecheck-notification/serialization.js.map: System error: net::ERR_FILE_NOT_FOUND
DevTools failed to load source map: Could not load content for chrome-extension://eppiocemhmnlbhjplcgkofciiegomcon/content/serialization.js.map: System error: net::ERR_FILE_NOT_FOUND
DevTools failed to load source map: Could not load content for chrome-extension://eppiocemhmnlbhjplcgkofciiegomcon/content/app-notification/serialization.js.map: System error: net::ERR_FILE_NOT_FOUND
log.ts:58 Process inventory: 162.48291015625 ms
log.ts:58 Inventory state update: 222.98681640625 ms
```

**Comment by nev-r** [comment]
i have no interest in fixing how tooltips place themselves but we can just delete it after the sheet looks nicer

**Comment by nev-r** [comment]
@RaySolo137 you can
- zoom out
- get a bigger browser window
- or (in beta today, in main sunday) click the consumables count number

**Comment by ahydronous** [comment]
https://github.com/DestinyItemManager/DIM/issues/8163 would be a possible roundabout way of solving this as you could list the items horizontally instead of vertically

**Comment by nev-r** [comment]
you could do that now though

**Comment by robojumper** [comment]
This was addressed in #8273.
